frontend/app/src/composables/links.ts

```diff
@@ -1,4 +1,4 @@
-export const useLinks = (url?: Ref<string>) => {
+export const useLinks = (url?: Ref<string | null>) => {
```

Comment:
this might need to become <string | undefined> after the change I suggested above.